src/main/flight/pid.h

```diff
@@ -245,6 +246,15 @@ typedef struct pidProfile_s {
     uint8_t ez_landing_threshold;           // Threshold stick position below which motor output is limited
     uint8_t ez_landing_limit;               // Maximum motor output when all sticks centred and throttle zero
     uint8_t ez_landing_speed;               // Speed below which motor output is limited
+
+    uint8_t chirp_lag_freq_hz;              // leadlag1Filter cutoff/pole to shape the excitation signal
+    uint8_t chirp_lead_freq_hz;             // leadlag1Filter cutoff/zero
+    uint16_t chirp_amplitude_roll;          // amplitude roll in degree/second
+    uint16_t chirp_amplitude_pitch;         // amplitude pitch in degree/second
+    uint16_t chirp_amplitude_yaw;           // amplitude yaw in degree/second
+    uint16_t chirp_frequency_start_deci_hz; // start frequency in units of 0.1 hz
+    uint16_t chirp_frequency_end_deci_hz;   // end frequency in units of 0.1 hz
+    uint8_t chirp_time_seconds;             // excitation time
```

Comment:
Maybe `#ifdef USE_CHIRP`. It will break PG upgrade, but chirp is quite specific feature, intended audience should know how to handle  switching chirp/nochirp version.